<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>801</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Банковские операции</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_Day">
               <property name="text">
                <string>День:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Month">
               <property name="text">
                <string>Месяц:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Year">
               <property name="text">
                <string>Год:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Income">
               <property name="text">
                <string>Доход:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Sender">
               <property name="text">
                <string>Отправитель:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Receuver">
               <property name="text">
                <string>Получатель:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Электронный</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLineEdit" name="lineEdit_Day"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_Month"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_Year"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_Summa"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_Sender"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_Receiver"/>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_Add">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_Refresh">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="spravka"/>
    <addaction name="quit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Начать</string>
   </property>
  </action>
  <action name="spravka">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
